mrmustard/physics/converters.py

```diff
@@ -59,3 +63,51 @@ def to_fock(rep: Representation, shape: Optional[Union[int, Iterable[int]]] = No
         array = [math.hermite_renormalized(A, b, c, shape) for A, b, c in zip(rep.A, rep.b, rep.c)]
         return Fock(math.astensor(array), batched=True)
     return rep
+
+
+def to_phase_space(
```

Comment:
This is circular import (we cannot import `State` from lab in physics). 
I think that this logic should be moved to the (existing, not implemented) `to_phase_space`) function of `State`